contracts/Factory.sol

```diff
@@ -27,27 +38,143 @@ contract Factory {
+            if (ins == 0x51) {continue;} // MLOAD
+            if (ins == 0x52) {continue;} // MSTORE
+            if (ins == 0x53) {continue;} // MSTORE8
+            if (ins == 0x54) {return 1;} // SLOAD
+            if (ins == 0x55) {return 2;} // SSTORE
+            if (ins == 0x56) {continue;} // JUMP
+            if (ins == 0x57) {continue;} // JUMPI
+            if (ins == 0x58) {continue;} // PC
+            if (ins == 0x59) {continue;} // MSIZE
+            if (ins == 0x5a) {continue;} // GAS
+            if (ins == 0x5b) {continue;} // JUMPDEST
 
             if (ins >= 0x60 && ins <= 0x7f) {
                 i += ins - 95;
             }
+
+            if (ins == 0x80) {continue;} // DUP1
+            if (ins == 0x81) {continue;} // DUP2
+            if (ins == 0x82) {continue;} // DUP3
+            if (ins == 0x83) {continue;} // DUP4
+            if (ins == 0x84) {continue;} // DUP5
+            if (ins == 0x85) {continue;} // DUP6
+            if (ins == 0x86) {continue;} // DUP7
+            if (ins == 0x87) {continue;} // DUP8
+            if (ins == 0x88) {continue;} // DUP9
+            if (ins == 0x89) {continue;} // DUP10
+            if (ins == 0x8a) {continue;} // DUP11
+            if (ins == 0x8b) {continue;} // DUP12
+            if (ins == 0x8c) {continue;} // DUP13
+            if (ins == 0x8d) {continue;} // DUP14
+            if (ins == 0x8e) {continue;} // DUP15
+            if (ins == 0x8f) {continue;} // DUP16
+
+            if (ins == 0x90) {continue;} // SWAP1
+            if (ins == 0x91) {continue;} // SWAP2
+            if (ins == 0x92) {continue;} // SWAP3
+            if (ins == 0x93) {continue;} // SWAP4
+            if (ins == 0x94) {continue;} // SWAP5
+            if (ins == 0x95) {continue;} // SWAP6
+            if (ins == 0x96) {continue;} // SWAP7
+            if (ins == 0x97) {continue;} // SWAP8
+            if (ins == 0x98) {continue;} // SWAP9
+            if (ins == 0x99) {continue;} // SWAP10
+            if (ins == 0x9a) {continue;} // SWAP11
+            if (ins == 0x9b) {continue;} // SWAP12
+            if (ins == 0x9c) {continue;} // SWAP13
+            if (ins == 0x9d) {continue;} // SWAP14
+            if (ins == 0x9e) {continue;} // SWAP15
+            if (ins == 0x9f) {continue;} // SWAP16
+
+            if (ins == 0xa0) {return 3;} // LOG0
+            if (ins == 0xa1) {return 4;} // LOG1
+            if (ins == 0xa2) {return 5;} // LOG2
+            if (ins == 0xa3) {return 6;} // LOG3
+            if (ins == 0xa4) {return 7;} // LOG4
+
+            if (ins == 0xf0) {return 8;} // CREATE
+            if (ins == 0xf1) {return 9;} // CALL
+            if (ins == 0xf2) {return 10;} // CALLCODE
+            if (ins == 0xf3) {continue;} // RETURN
+            if (ins == 0xf4) {return 11;} // DELEGATECALL
+            if (ins == 0xf5) {return 12;} // CREATE2
+            if (ins == 0xfa) {continue;} // STATICCALL
+            if (ins == 0xfd) {continue;} // REVERT
```

Comment:
Do we allow REVERT? I'm not sure what would happen in the context of a nested delegate in a procedure call.

Reply:
Yeah, we've always allowed revert, it's necessary for normal operation of contracts. The failure of contracts is handled, just as if there was an invalid operation.